src/Widgets/PreferencesDialog.vala

```diff
@@ -19,6 +19,45 @@
 * Authored by: Felipe Escoto <felescoto95@hotmail.com>
 */
 
+/**
+ *  The purpose of this class is to easily create a new Switchbox for the
+ *  PreferencesDialog class.
+ */
+private class ENotes.PreferencesDialogSwitchBox {
+
```

Comment:
Are you cool with me just keeping the class? I will add it to the PreferencesDialog class. I do not think will harm anything.